zip-zcash-grants-fund.rst

```diff
@@ -0,0 +1,330 @@
+charitable, educational, and scientific activities within the meaning of
+Section 501(c)(3). This approach will ensure continuous funding through a
+structured grant application process, fostering the ongoing development and
+decentralization of the Zcash ecosystem.
+
+Requirements
+============
+
+The Grants Fund should encourage decentralization of the work and funding, by
+supporting new teams dedicated to Zcash. The Grants Fund should maintain the
+existing teams and capabilities in the Zcash ecosystem, unless and until
+concrete opportunities arise to create even greater value for the Zcash ecosystem.
+
+Grant funding decisions should be based, to the extent feasible, on inputs from
+domain experts and pertinent stakeholders.
+
+The Grants Fund mechanism should not modify the monetary emission curve (and in
+particular, should not irrevocably burn coins).
+
+In case the value of ZEC jumps, the Grant Fund recipients should not wastefully
+use excessive amounts of funds. Conversely, given market volatility and eventual
+halvings, it is desirable to create rainy-day reserves.
+
+The Grants Fund mechanism should not reduce users' financial privacy or security.
+In particular, it should not cause them to expose their coin holdings, nor
+cause them to maintain access to secret keys for much longer than they would
+otherwise. (This rules out some forms of voting, and of disbursing coins to
+past/future miners.)
+
+The new Grants Fund system should be simple to understand and realistic to
+implement. In particular, it should not assume the creation of new mechanisms
+(e.g., election systems) or entities (for governance or development) for its
+execution; but it should strive to support and use these once they are built.
+
+Comply with legal, regulatory, and taxation constraints in pertinent
+jurisdictions.
+
+
+Non-requirements
+================
+
+General on-chain governance is outside the scope of this proposal.
+
+Rigorous voting mechanisms (whether coin-weighted, holding-time-weighted or
+one-person-one-vote) are outside the scope of this proposal, though there is
+prescribed room for integrating them once available.
+
+
+Specification
+=============
+
+Consensus changes implied by this specification are applicable to the
+Zcash Mainnet. Similar (but not necessarily identical) consensus changes
+SHOULD be applied to the Zcash Testnet for testing purposes.
+
+
+Dev Fund allocation
+-------------------
+
+Starting at the second Zcash halving in 2024, for 2 years,
+20% of the block subsidy of each block SHALL be allocated to a "Grants Fund" that
+will be 100% allocated to "Zcash Community Grants" for funding grants that support projects
+and initiatives aligned with Zcash's mission.
+
+The fund flow will be implemented
+at the consensus-rule layer, by sending the corresponding ZEC to the designated
+address(es) for each block. This Dev Fund will end 2 years after the second halving (unless

```

Comment:
Is this required to be a multisig? Are there any specified procedures to respond in the case of potential abuse or theft of funds or of key compromise?